scss/detail.scss

```diff
@@ -0,0 +1,40 @@
+$gray : #828282;
+
+.book {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  flex-direction: column;
+  padding: 15px;
+}
+
+.row {
+  padding: 18px 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.title {
+  font-size: 32px;
+  font-weight: bold;
+  flex-direction: row;
+}
+
+.gender {
+  color: $gray;
+  font-size: 24px;
+  font-weight: bold;
+  padding: 0 8px;
+}
+
+.author,
+.editorial,
+.year,
+.description {
+  font-size: 20px;
+  line-height: 24px;
+}
```

Comment:
Good idea, but we could use define a class for different text types and reuse in our HTML file. Like `text-1` and place it in a `texts.scss` file to import it afterwards